magni_gazebo/config/magni_controllers.yaml

```diff
@@ -0,0 +1,43 @@
+
+ubiquity_joint_publisher:
+  type: "joint_state_controller/JointStateController"
+  publish_rate: 50
+
+ubiquity_velocity_controller:
+  type: "diff_drive_controller/DiffDriveController"
+  left_wheel  : 'left_wheel_joint'
+  right_wheel : 'right_wheel_joint'
+  publish_rate: 50.0
+  pose_covariance_diagonal : [0.001, 0.001, 1000000.0, 1000000.0, 1000000.0, 1000.0]
```

Comment:
Please use the same covariance as the real robot https://github.com/UbiquityRobotics/magni_robot/blob/indigo-devel/magni_bringup/param/base.yaml#L20